### ISSUE TITLE
KBFS not found windows 10

### ISSUE THREAD

**Issue by evinhernandez** [keybase/client#6543]
[write something useful and descriptive here]

my log id: 55825c43e7eab6b25fb8621c

I have to restart my entire windows 10 system for kbfs to be found again. sometimes it just crashes after going to sleep or if i need to kill the program and start it up again it just stays at loading .

**Comment by maxtaco** [comment]
cc: @zanderz

**Comment by plttn** [comment]
@evinhernandez Do you have a Surface? Or another device with InstantGo?

see #5215

**Comment by evinhernandez** [comment]
@plttn Yes i have a surface book.

**Comment by plttn** [comment]
Yup, currently as far as I can tell there's no resolution to this.

**Comment by evinhernandez** [comment]
so my work around would be to just reboot on my windows machine ?

**Comment by plttn** [comment]
Yup.

**Comment by evinhernandez** [comment]
@plttn Damn windows !!!! thank you for your help

**Comment by taruti** [comment]
See https://github.com/keybase/client/issues/5215 we are working on getting this resolved.
